TFUtil.py

```diff
@@ -4850,7 +4850,7 @@ def __init__(self, include_paths=(), ld_flags=(), **kwargs):
     ld_flags = list(ld_flags)
     if have_min_tf_version((1, 4)):
       # https://github.com/tensorflow/tensorflow/issues/13607
-      ld_flags += ["-L%s" % tf.sysconfig.get_lib(), "-ltensorflow_framework"]
+      ld_flags += tf.sysconfig.get_link_flags()
```

Comment:
Be careful here. I think `get_link_flags()` was introduced only later (later than TF 1.4). So maybe we need another check `have_min_tf_version((1, 14))` (or whatever version) for that.